### PR TITLE
Remove regex-pcre-builtin

### PR DESCRIPTION
**Pull request by theotherjimmy**

### CODE REVIEW

**Comment by fosskers** [comment]
Is this a [WIP]? I feel like there is more `regex` use than that around.

File: src/Aura/MakePkg.hs [hunk 1]
```diff
@@ -31,27 +31,26 @@ module Aura.MakePkg
...
 import           Utilities
+import           Aura.Shell
 
 ---
 
 type User = Text
 
+findPkgFile :: FilePath -> Sh [FilePath]
+findPkgFile = findDirFilterWhen (pure . const False) (pure . matches)
+  where matches fp = ["xz", "tar", "pkg"] `isPrefixOf` reverse (extensions fp)
```
Comment (fosskers) [inline src/Aura/MakePkg.hs]: Will this also find the `.sig` file, if one is present?

**Comment by theotherjimmy** [comment]
I was just trying to remove the regex-pcre-builtin library.

File: src/Aura/MakePkg.hs (hunk 1 above)
Comment (theotherjimmy) [inline src/Aura/MakePkg.hs]: I don't think so. Also, I don't think that the old code did that. OTOH, I gooffed and that should be "src" and not "pkg".

**Comment by fosskers** [comment]
Great, let me know when I can take a final look at this :+1:

**Comment by theotherjimmy** [comment]
Should be good to go. It's not a huge change after all.

**Comment by theotherjimmy** [comment]
Tested works for normal, did not test with source build, I assume that it does not work for source builds.

**Comment by theotherjimmy** [comment]
Tested source builds. works.

**Comment by theotherjimmy** [comment]
@fosskers, I think it's ready.

**Comment by fosskers** [comment]
Great I'll take a look.

**Comment by theotherjimmy** [comment]
Hey @fosskers, What's the verdict?

**Comment by fosskers** [comment]
Hm, my builds are bailing. 

```
sudo .stack-work/dist/x86_64-linux/Cabal-1.22.5.0/build/aura/aura -Ax packer
[sudo] password for colin: 
aura >>= Determining dependencies...
aura >>= AUR Packages:
packer
aura >>= Continue? [Y/n] 
aura >>= Building `packer`...
x packer/
x packer/.AURINFO
x packer/.SRCINFO
x packer/PKGBUILD
==> Making package: packer 20150808-1 (Mon Mar 14 18:01:49 PDT 2016)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Cloning packer git repo...
Cloning into bare repository '/home/colin/code/haskell/aura/packer/packer'...
==> Validating source files with md5sums...
    packer ... Skipped
==> Extracting sources...
  -> Creating working copy of packer git repo...
Cloning into 'packer'...
done.
==> Starting pkgver()...
==> Entering fakeroot environment...
==> Starting package()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issue...
==> Creating package "packer"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: packer 20150808-1 (Mon Mar 14 18:02:07 PDT 2016)
loading packages...
resolving dependencies...
warning: packer-20150808-1 is up to date -- reinstalling
looking for conflicting packages...

Packages (1) packer-20150808-1

Total Installed Size:  0.05 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n] 
aura: 
Ran commands: 
pwd
cd /var/cache/pacman/pkg/
cd /home/colin/code/haskell/aura
pacman -U /home/colin/code/haskell/aura/packer/packer-20150808-1-any.pkg.tar.xz
which pacman

Exception: error running: pacman -U /home/colin/code/haskell/aura/packer/packer-20150808-1-any.pkg.tar.xz
exit status: 1
```

**Comment by fosskers** [comment]
Seems to be both. On `aura-1.4`, built packages are dumped into `./` and then not installed. `makepkg` succeeds, and then aura exits without trying to install the built package.

**Comment by theotherjimmy** [comment]
Yes. I'm working on a PR that fixes it.

**Comment by theotherjimmy** [comment]
@fosskers, this is ready for review.

**Comment by fosskers** [comment]
@theotherjimmy I'm going to borrow these changes back into master, so we can dump the old dep for `1.3.7`.

**Comment by theotherjimmy** [comment]
Good plan. :+1:

**Comment by fosskers** [comment]
Actually it turns out the offending `regex-pcre-builtin` is still in `[haskell-core]`, so I'm keeping it for now.

**Pull request merged**